### PR TITLE
Multi Cluster and Auto query yml support

### PR DESCRIPTION
**Pull request by sharmay**
- Add support for multi cluster on single server
  - obsolete jobname
  - Add pgnodes, osnodes new variables
  - New OSResource dashboard for osnodes
- Auto create query yml at service start
  - Added new variable in sysconfig QUERY_FILE_LIST
  - Added code to .service to auto create target query yml file

### CODE REVIEW

**Comment by sharmay** [comment]
systemd service files Tested on CentOS 7 only.

File: prometheus/auto.d/ReplicaDB.yml.example [hunk 1]
```diff
@@ -1,3 +1,4 @@
 - targets: [ "10.123.202.13:9100", "10.123.202.13:9187" ]
   labels:
     job: "Staging"
```
Comment (keithf4) [inline prometheus/auto.d/ReplicaDB.yml.example]: Calling this "staging" in the replica file is a bit confusing. Change job name to "Replica"?

Also, need to remove the node_exporter target from here.

File: prometheus/auto.d/ReplicaOS.yml.example [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- targets: [ "10.123.202.13:9100", "10.123.202.13:9187" ]
```
Comment (keithf4) [inline prometheus/auto.d/ReplicaOS.yml.example]: Need to remove postgres exporter target from here and change name to "Replica"

File: exporter/postgres/sysconfig.postgres_exporter_pg11 [hunk 3]
```diff
@@ -3,7 +3,13 @@
 # --web.listen-address: change '0.0.0.0' to the network IP assigned to this system if necessary, otherwise it will listen on any IP. Change port as necessary if running multiple instances.
 # --extend.query-path: location of file containing custom queries to run. Location below is one recommended by crunchy setup steps.
 # DATA_SOURCE_NAME: psql connection string. set the database that the exporter will connect to. Default is `postgres`.
+# QUERY_FILE_LIST: space delimitted yml files to use as queries.yml
 
+PGBACKREST_INFO_THROTTLE_MINUTES=10
 OPT="--web.listen-address=0.0.0.0:9187 --extend.query-path=/etc/postgres_exporter/11/queries.yml --disable-default-metrics"
 DATA_SOURCE_NAME="postgresql://ccp_monitoring@localhost:5432/postgres?sslmode=disable"
+# For global / cluster metric
+QUERY_FILE_LIST="/etc/postgres_exporter/11/queries_common.yml /etc/postgres_exporter/11/queries_pg11.yml /etc/postgres_exporter/11/queries_per_db.yml /etc/postgres_exporter/11/queries_bloat.yml /etc/postgres_exporter/11/queries_backrest.yml"
```
Comment (keithf4) [inline exporter/postgres/sysconfig.postgres_exporter_pg11]: Recommend leaving out bloat & backrest yml files by default. Same for all sysconfig files.

Comment (sharmay) [reply exporter/postgres/sysconfig.postgres_exporter_pg11]: Fixed in https://github.com/CrunchyData/pgmonitor/pull/89/commits/2cf98aa2d8ae00d6c8c10707d86194af6cc0adf7

File: prometheus/auto.d/ReplicaOS.yml.example (hunk 2 above)
Comment (sharmay) [reply prometheus/auto.d/ReplicaOS.yml.example]: Fixed in https://github.com/CrunchyData/pgmonitor/pull/89/commits/2cf98aa2d8ae00d6c8c10707d86194af6cc0adf7

File: prometheus/auto.d/ReplicaDB.yml.example (hunk 1 above)
Comment (sharmay) [reply prometheus/auto.d/ReplicaDB.yml.example]: Fixed in https://github.com/CrunchyData/pgmonitor/pull/89/commits/2cf98aa2d8ae00d6c8c10707d86194af6cc0adf7

**Pull request merged**